src/sentry/utils/eventuser.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from typing import Optional, Tuple
+
+from snuba_sdk import Column, Condition, Entity, Op, Query, Request
+
+from sentry.eventstore.models import Event
+from sentry.snuba.dataset import Dataset, EntityKey
+from sentry.utils.snuba import raw_snql_query
+
+logger = logging.getLogger(__name__)
+
+REFERRER = "sentry.utils.eventuser"
+
+
+@dataclass
+class EventUser:
```

Comment:
You'll need to register this with the `EventUser` serializer to be able to return responses